package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "warehouse-models",
-  "version": "5.9.1",
+  "name": "warehouse-models-dynamo",
+  "version": "6.0.0-0",
```

Comment:
Where did the '-0' come from? Also, shouldn't we be doing the version bump in a separate commit?

Reply:
@jcrugzz  I thought we were supposed to add `-0` to indicate it as a pre-release for some reason?

Reply:
@wjarvis1 @cfellin1 In the other models I did this, but im considering we just publish this as a new major and leverage the npm aliasing method to install 2 versions of the same package for migration purposes. and yes we should do versioning after its merged.